include/wil/com.h

```diff
@@ -1846,46 +1858,90 @@ namespace wil
         return SUCCEEDED(com_raw_ptr(wistd::forward<T>(ptrSource))->QueryInterface(IID_PPV_ARGS(&agileObject)));
     }
 
-    /** constructs a COM object using the class as the identifier (that has an associated CLSID) on a specific interface or IUnknown.*/
-    template<typename Class, typename Interface = IUnknown, typename error_policy = err_exception_policy>
-    wil::com_ptr_t<Interface, error_policy> CoCreateInstance(DWORD dwClsContext = CLSCTX_INPROC_SERVER)
+    /** constructs a COM object using an CLSID on a specific interface or IUnknown.*/
+    template<typename Interface = IUnknown, typename error_policy = err_exception_policy>
+    wil::com_ptr_t<Interface, error_policy> CoCreateInstance(REFCLSID rclsid, DWORD dwClsContext = CLSCTX_INPROC_SERVER)
     {
         wil::com_ptr_t<Interface, error_policy> result;
-        error_policy::HResult(::CoCreateInstance(__uuidof(Class), nullptr, dwClsContext, IID_PPV_ARGS(&result)));
+        error_policy::HResult(::CoCreateInstance(rclsid, nullptr, dwClsContext, IID_PPV_ARGS(&result)));
         return result;
     }
 
+    /** constructs a COM object using the class as the identifier (that has an associated CLSID) on a specific interface or IUnknown. */
+    template<typename Class, typename Interface = IUnknown, typename error_policy = err_exception_policy>
+    wil::com_ptr_t<Interface, error_policy> CoCreateInstance(DWORD dwClsContext = CLSCTX_INPROC_SERVER)
+    {
+        return CoCreateInstance<Interface, error_policy>(__uuidof(Class), dwClsContext);
+    }
+
+    /** constructs a COM object using an CLSID on a specific interface or IUnknown. */
+    template<typename Interface = IUnknown>
+    wil::com_ptr_failfast<Interface> CoCreateInstanceFailFast(REFCLSID rclsid, DWORD dwClsContext = CLSCTX_INPROC_SERVER)
+    {
+        return CoCreateInstance<Interface, err_failfast_policy>(rclsid, dwClsContext);
+    }
+
     /** constructs a COM object using the class as the identifier (that has an associated CLSID) on a specific interface or IUnknown. */
     template<typename Class, typename Interface = IUnknown>
     wil::com_ptr_failfast<Interface> CoCreateInstanceFailFast(DWORD dwClsContext = CLSCTX_INPROC_SERVER)
     {
-        return CoCreateInstance<Class, Interface, err_failfast_policy>(dwClsContext);
+		return CoCreateInstanceFailFast<Interface>(__uuidof(Class), dwClsContext);

```

Comment:
fix indentation

Reply:
Thanks, resolved. It would be a good idea to add a `.editorconfig` file to the repo so that text editors automatically pick up the desired indentation style.